chatmaild/src/chatmaild/rm_accounts.py

```diff
@@ -0,0 +1,54 @@
+import crypt
+import json
+import logging
+import os
+import sys
+import time
+import shutil
+from pathlib import Path
+from socketserver import (
+    StreamRequestHandler,
+    ThreadingMixIn,
+    UnixStreamServer,
+)
+
+from .config import Config, read_config
+from .database import Database
+
+
+def remove_users(db: Database, cutoff_date: int):
+    db.connect()
+    try:
```

Comment:
the typical pattern is rather "with db.write_transaction()" like in other places. You don't need explicit committing and finally-clauses then.